### PR TITLE
Update the serialization of variants

### PR DESCRIPTION
**Pull request by wachterjohannes**

### CODE REVIEW

File: bin/create_node_symlink.php [hunk 1]
```diff
@@ -1,11 +1,21 @@
 <?php
 
+declare(strict_types=1);
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
```
Comment (niklasnatter) [inline bin/create_node_symlink.php]: 🤔

**Comment by wachterjohannes** [comment]
@alexander-schranz @nnatter we have to fix the tests in a follow up PR

**Comment by alexander-schranz** [comment]
@wachterjohannes looks for me the `FOSOAuthServerBundle` is not compatible with the newest doctrine version.

**Review by alexander-schranz — APPROVED** [review]

**Comment by wachterjohannes** [comment]
@alexander-schranz as the tests on master also fails - we should merge this and take a look at this deeper in a follow up

**Pull request merged**